api/src/prescription/learner-management/domain/validators/common-organization-learner-validator.js

```diff
@@ -0,0 +1,24 @@
+import JoiDate from '@joi/date';
+import BaseJoi from 'joi';
+const Joi = BaseJoi.extend(JoiDate);
+
+const validationConfiguration = { allowUnknown: true, abortEarly: false };
+
+const validateCommonOrganizationLearner = function (commonOrganizationLearner, validationRules) {
```

Comment:
**nitpick** : Je suis un peu embêté qu'on l'appelle commonOrganizationLearner alors que c'est pas un objet de ce model.

Reply:
Avec le changement fait dans le fichier `CommonOrganizationLearnerSet` 
`addLearner(learnerAttributes) {
    const learner = new CommonOrganizationLearner(learnerAttributes);
    this.#validateRules(learner);
    this.#learners.push(learner);
  }`
  On passe maintenant un commonOrganizationLearner